### PR TITLE
Feature/python -- Fixes #4

### LINKED ISSUE #4: Add Python implementation.
It would be great to have a Python implementation of the metaparticle/package library.

### PR DESCRIPTION
**Pull request by willnewby**
Comments and feedback welcome.

### CODE REVIEW

File: python/main.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        raise Exception('Unknown buildSpec {}'.format(buildSpec))
+
+
+def get_name(options):
+    try:
+        return options['name']
+    except AttributeError:
+        pass
+
+    workdir = os.getcwd().split('/')
```
Comment (mrallen1) [inline python/main.py]: Is there a reason you wouldn't want to use `os.path.basename(os.getwd())` as a default here?

Comment (willnewby) [reply python/main.py]: Great call, didn't realize you could do that. I'll update.

File: python/main.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
+import functools
```
Comment (mrallen1) [inline python/main.py]: Just wondering now if naming this file `metaparticle` would make more sense.  The import would be something like

`from metaparticle import containerize`

Comment (willnewby) [reply python/main.py]: Oh! Also good. I'll fix that too.

**Pull request merged**